portals/publisher/source/src/app/components/Apis/Details/Policies/OperationButton.tsx

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { FC } from 'react';
+import Button from '@material-ui/core/Button';
+import { makeStyles } from '@material-ui/core/styles';
+import Utils from 'AppData/Utils';
+
+interface OPProps {
+    operation: any;
+    verb: any;
+}
+
+const OperationButton: FC<OPProps> = ({ operation, verb }) => {
+    const useStyles = makeStyles((theme: any) => {
```

Comment:
Take the `makeStyles` calls outside the component, [`makeStyles`](https://v4.mui.com/styles/api/#makestyles-styles-options-hook) is a higher-order hook, which returns a hook, When it's inside a component it will create a new hook everytime component gets rendered.
Check the [docs](https://v4.mui.com/styles/api/#makestyles-styles-options-hook) for usage example

Reply:
fixed with: https://github.com/wso2/apim-apps/pull/177/commits/3be4d1db628b1a5a5e420682698a04bcd29b7f74